gateway-server/src/main/java/org/apache/knox/gateway/services/security/impl/ZookeeperRemoteAliasService.java

```diff
@@ -237,6 +238,13 @@ public void removeAliasForCluster(final String clusterName, final String alias)
         }
     }
 
+    @Override
+    public void removeAliasesForCluster(String clusterName, Set<String> aliases) throws AliasServiceException {
```

Comment:
Not related to review but just a side note: we should make some calls to RemoteAliasService non blocking, it would help a lot.

Reply:
I'm not sure what you mean here. Can you elaborate?

Reply:
Out of scope for this change IMO

Reply:
yup, this was just a note :)